src/fetchcode/package.py

```diff
@@ -362,6 +372,123 @@ def get_gnu_data_from_purl(purl):
     )
 
 
+@router.route("pkg:cocoapods/.*")
+def get_cocoapods_data_from_purl(purl):
+    """
+    Generate `Package` object from the `purl` string of cocoapods type
+    """
+    logging.basicConfig(
```

Comment:
I am not sure we want logging in the library, but I am sure we do not want logging to be enabled and configured here at all.

Reply:
Confirming that my added logging has been removed.